include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -286,7 +286,10 @@ namespace internal
 
         Assert(ptr != dof_handler.hp_object_fe_indices[d].begin() +
                         dof_handler.hp_object_fe_ptr[d][obj_index + 1],
-               ExcNotImplemented());
+               ExcMessage(
+                 "You are requesting from this entity an "
+                 "active_fe_index that is not assigned to any of the cells "
+                 "connected to this entity."));
```

Comment:
```suggestion
                 "You are requesting an active_fe_index that is not assigned "
                 "to any of the cells connected to this entity."));
```

Reply:
I have changed the text. Good to have a native speaker in the team :+1: